src/components/Transaction/Buttons.tsx

```diff
@@ -8,17 +8,25 @@ export const TransactionButtons = ({
   nextDisabled,
   backText,
   nextText,
+  backType,
+  nextType,
```

Comment:
curious - why do these need to be passed as props instead of hardcoded in this context? is the next button not always `submit` and the back button not `button`

Judging from you adding `onClickNext` - i'm sure i'm not thinking of some instance of needing this in one of our apps

Reply:
Yeah I think you are right, we can make this less versatile and more tailored to our use case, so it encourages using the buttons in a form with the submit action